### ISSUE TITLE
SyntaxError

### ISSUE THREAD

**Issue by ghost** [socsieng/sendkeys-macos#1]
const sendkeys = require('sendkeys-macos')

sendkeys -c "<c:v:command>"
sendkeys -c "enter"

with the quotation marks around <c:v:command> it says unexpected string (syntaxerror) and without them it says unexpected token ":" (syntaxerror)

**Comment by socsieng** [comment]
It looks like a limitation with Automator handling emoji characters.

Executing the following results in `aa` being sent to Notes:

```sh
osascript <<EOF
tell application "Notes" to activate
tell application "System Events"
    keystroke "😀"
end tell
EOF
```

**Comment by ghost** [comment]
oops! sorry!
its actually <c: v : command>
without spaces obviously. I just needed to paste something and then press enter!
I also need to do all of this from inside a js file (run using node from terminal)

**Comment by socsieng** [comment]
I didn't plan for the package to be used in nodejs applications originally but I've fixed that now (as of v0.2.0).

The way to do that is:

```js
const sendKeys = require('sendkeys-macos');

sendKeys(
  // name of application to send keystrokes to
  'Notes',
  // keystrokes to send
  '<c:v:command><c:enter>',
  {
    // delay before sending keystrokes,
    // need to give the application a chance to activate before sending keystrokes
    initialDelay: 1,
  });

```
